### DISCUSSION TITLE
Edit edge label of newly created edge

### DISCUSSION THREAD

**Discussion by claraisrael** [xyflow/xyflow#3562] [Q&A]
Hello, I'm trying edit/write the label of the edges when connecting two nodes.

Does react flow support this functionality? would be super useful

**Comment by bcakmakoglu** [comment]
How is this supposed to look like? Creating a connection is done by dragging it from handle to handle, so typing something during that isn't gonna work out.

You can, however, use a label on an existing edge and make it editable using the [`EdgeLabelRenderer`](https://reactflow.dev/api-reference/components/edge-label-renderer) and putting an input field into it.
